content/20.pharmaceuticals.md

```diff
@@ -159,17 +182,58 @@ Angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin II receptor blo
 In the United States, for example, they are prescribed well over 100,000,000 times annually [@doi:10.18553/jmcp.2010.16.9.671].
 Prior to the COVID-19 pandemic, the relationship between ACE2, ACEIs, and SARS had been considered as possible evidence that ACE2 could serve as a therapeutic target [@doi:10.1016/j.tips.2004.04.001], and the connection had been explored through _in vitro_ and molecular docking analysis [@doi:10.1161/01.HYP.0000146120.29648.36] but ultimately was not pursued clinically [@doi:10/d85vmw].
 Data from some animal models suggest that several, but not all, ACEIs and several ARBs increase ACE2 expression in the cells of some organs [@doi:10.1093/cvr/cvaa097], but clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications [@doi:10.1093/europace/euw246].
+
 In this case, rather than introducing ARBs/ACEIs, a number of analyses have investigated whether discontinuing use affects COVID-19 outcomes.
 An initial observational study of the association of exposure to ACEIs or ARBs with outcomes in COVID-19 was retracted from the _New England Journal of Medicine_ [@doi:10.1056/NEJMoa2007621] due to concerns related to data availability [@doi:10.1056/nejmc2021225].
-As randomized trials have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes [@doi:10/fvgt; @doi:10.1001/jama.2020.25864] (Appendix XX).
+As randomized trials have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes [@doi:10/fvgt; @doi:10.1001/jama.2020.25864] (Appendix A).
 Thus, once again, despite a potential mechanistic association with the pathology of SARS-CoV-2 infection, these medications were not found to influence the trajectory of COVID-19 illness.
 However, because the medications were so widespread, clinical research was able to evaluate their efficacy very quickly.
+This feasbility presents a double-edged sword, because HCQ/CQ were incorporated into SOC in many countries before being removed once the evidence indicated its potential to harm COVID-19 patients [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].
```

Comment:
```suggestion
This feasibility presents a double-edged sword, because HCQ/CQ were incorporated into SOC in many countries before being removed once the evidence indicated its potential to harm COVID-19 patients [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].
```